prac_06/used_cars.py

```diff
@@ -0,0 +1,43 @@
+"""CP1404/CP5632 Practical - Client code to use the Car class."""
+# Note that the import has a folder (module) in it.
+
+from prac_06.car import Car
+
+
+def main():
+    """Demo test code to show how to use car class."""
+    my_car = Car("Car", 180)
+    my_car.drive(30)
+    print("fuel =", my_car.fuel)
+    print("odo =", my_car.odometer)
+    print(my_car)
+
+    print("Car {}, {}".format(my_car.fuel, my_car.odometer))
+    print("Car {self.fuel}, {self.odometer}".format(self=my_car))
+
+    """1. Create a new Car object called "limo" that is initialised with 100 units of fuel."""
+    limo = Car('Limo', 100)
```

Comment:
As mentioned these docstrings may not adhere to PEP standards. Elaborating on code is better performed by comments. Even so, this can sometimes add too much "noise" to the program.